test/built-ins/Temporal/PlainTime/prototype/since/plaintime-propertybag-no-time-units.js

```diff
@@ -0,0 +1,18 @@
+// Copyright (C) 2021 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-temporal.plaintime.prototype.since
+description: Missing time units in property bag default to 0
+includes: [temporalHelpers.js]
+features: [Temporal]
+---*/
+
+const instance = new Temporal.PlainTime(1, 0, 0, 0, 0, 1);
+
+const props = {};
+assert.throws(TypeError, () => instance.since(props), "TypeError if at least one property is not present");
```

Comment:
"if no properties are present"?

Reply:
I copied the assertion message from the existing tests, so I'll update it there as well.